### PR TITLE
NO-ISSUE: Update Makefile target dependencies to include deps-update

### PR DESCRIPTION
**Pull request by donpenney**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -135,7 +135,7 @@ run: manifests generate fmt vet binary ## Run a controller from your host.
 # (i.e. docker build --platform linux/arm64). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
-docker-build: ## Build docker image with the manager.
+docker-build: manifests generate fmt vet ## Build docker image with the manager.
```
Comment (alegacy) [inline Makefile:138]: Should this also include `build`?

Comment (donpenney) [reply Makefile:138]: No, `build` is for building outside a container, so isn't needed for the `docker-build`

Comment (alegacy) [reply Makefile:138]: Oh right, sorry ... forgot that's handled in the Dockerfile.

**Comment by donpenney** [comment]
PR has been rebased

**Pull request merged**